src/components/CommentField.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { useRef, useState, FormEvent, useEffect } from 'react';
+import { EnrichedActivity } from 'getstream';
+
+import { Avatar } from './Avatar';
+import { Button } from './Button';
+import { Textarea, TextareaProps } from './Textarea';
+import { inputValueFromEvent } from '../utils';
+import { useTranslationContext, FeedManager } from '../Context';
+import { DefaultAT, DefaultUT } from '../Context/StreamApp';
+
+export type CommentFieldProps<UT extends DefaultUT = DefaultUT, AT extends DefaultAT = DefaultAT> = {
+  activity: EnrichedActivity<UT, AT>;
+  onAddReaction: FeedManager<UT, AT>['onAddReaction'];
+  image?: string;
+  kind?: string;
+  onSuccess?: () => void;
+  placeholder?: string;
+  trigger?: TextareaProps['trigger'];
+};
+
+export const CommentField = <UT extends DefaultUT = DefaultUT, AT extends DefaultAT = DefaultAT>({
+  activity,
+  onAddReaction,
+  onSuccess,
+  image,
+  placeholder,
+  trigger,
+}: CommentFieldProps<UT, AT>) => {
+  const { t } = useTranslationContext();
+  const textareaReference = useRef<HTMLTextAreaElement>();
+  const [text, setText] = useState<string>();
+
+  const handleFormSubmit = async (event: FormEvent<HTMLFormElement> | KeyboardEvent) => {
+    event.preventDefault();
+
+    if (!text) return;
+
+    try {
+      await onAddReaction('comment', activity, {
+        text,
+      });
```

Comment:
```suggestion
      await onAddReaction('comment', activity, { text });
```

Reply:
Not sure why linter allowed me to commit this. 🤔